examples/pwm.rs

```diff
@@ -26,7 +27,7 @@ fn main() -> ! {
 
     let mut pwm = dp.TIM1.pwm(pin, 100.Hz(), &mut rcc);
 
-    pwm.set_duty(pwm.get_max_duty() / 2);
+    let _ = pwm.set_duty_cycle(pwm.max_duty_cycle() / 2);
```

Comment:
Would it even make sense to use [set_duty_cycle_percent](https://docs.rs/embedded-hal/latest/embedded_hal/pwm/trait.SetDutyCycle.html#method.set_duty_cycle_percent) to set 50% duty?

Reply:
Good call, I had forgotten it was a thing